packages/cannon-worker-api/src/cannon-worker-api.ts

```diff
@@ -42,6 +42,14 @@ export class CannonWorkerAPI extends EventEmitter {
     this.postMessage({ op: 'setGravity', props: value })
   }
 
+  get frictionGravity(): Triplet | undefined {
```

Comment:
None of our interface is optional, it should always return a Triplet.

Reply:
In this case, a possible value for `frictionGravity` is `undefined`.

https://pmndrs.github.io/cannon-es/docs/classes/World.html#frictionGravity

Reply:
Right, thanks Isaac! @bjornstar I could return an empty array here if always returning a value is important - but keeping the worker api signature consistent with the `World` interface seems preferable to me.